src/js/login/reducers/login.js

```diff
@@ -1,15 +1,31 @@
 import _ from 'lodash/fp';
 
-import { UPDATE_LOGGEDIN_STATUS, UPDATE_LOGIN_URL, LOG_OUT } from '../actions';
+import { UPDATE_LOGGEDIN_STATUS, UPDATE_LOGIN_URL, LOG_OUT, UPDATE_SEARCH_HELP_USER_MENU } from '../actions';
 
 const initialState = {
-  currentlyLoggedIn: false,
+  currentlyLoggedIn: true,
   loginUrl: {
     first: null,
     third: null
   },
+  utilitiesMenuIsOpen: {
+    search: false,
+    help: false,
+    account: false
+  }
 };
 
+function closeAllMenus(menuState) {
+  const menus = menuState.utilitiesMenuIsOpen;
+  let menu;
+
+  for (menu in menus) {
+    if (menus.hasOwnProperty(menu)) {
```

Comment:
Could avoid checking for this by iterating with `Object.keys(menus).forEach(...)`.

But since this is to overwrite the state, if we want to keep things "immutable", it's probably better to just explicitly set it with `_.set('utilitiesMenuIsOpen', initialState.utilitiesMenuIsOpen, menuState);`.

Or to apply it directly to the case below, it could succinctly be refactored as `_.set('utilitiesMenuIsOpen', { ...initialState.utilitiesMenuIsOpen, [action.menu]: action.isOpen }, state);`.